File: rasterizer-dotnet-examples-csharp/PngRgbaTransparency.cs
using ceTe.DynamicPDF.Rasterizer;


namespace rasterizer_dotnet_csharp
{
    class PngRgbaTransparency
    {
        internal static void Run()
        {
            PdfRasterizer rasterizer = new PdfRasterizer(Program.GetResourcePath("pdf-with-transparency.pdf"));
            FixedImageSize fixedImageSize = new FixedImageSize(595, 841);
            PngImageFormat oPngImageFormat = new PngImageFormat(PngColorFormat.RgbA);
            rasterizer.Draw(Program.GetOutputDocPath("RgbaTransparency-png-output.png"), oPngImageFormat, fixedImageSize);
        }
    }
}

File: rasterizer-dotnet-vb/PngRgbaTransparency.vb
Imports ceTe.DynamicPDF.Rasterizer

Class PngRgbaTransparency

    Friend Shared Sub Run()
        Dim rasterizer As New PdfRasterizer(Program.GetResourcePath("pdf-with-transparency.pdf"))
        Dim fixedImageSize As New FixedImageSize(595, 841)
        Dim oPngImageFormat As New PngImageFormat(PngColorFormat.RgbA)
        rasterizer.Draw(Program.GetOutputDocPath("RgbaTransparency-png-output.png"), oPngImageFormat, fixedImageSize)
    End Sub

End Class

